example/src/components/RetryPolicyConfiguration.tsx

```diff
@@ -122,6 +157,52 @@ const RetryPolicyConfiguration = (props: RetryPolicyConfigurationProps) => {
                     />
                 </>
             )}
+
+            {props.policyType && (
+                <>
+                    <Input
+                        placeholder="Retry Status Codes"
+                        disabled={true}
+                        style={{ fontWeight: "bold", fontSize: 17, opacity: 1 }}
+                        containerStyle={{ height: 50 }}
+                        inputContainerStyle={{
+                            borderColor: "rgba(255,255,255,0)",
+                        }}
+                    />
+
+                    <Input
+                        value={statusCodesText}
+                        onChangeText={setStatusCodesText}
+                        onBlur={updateStatusCodes}
+                    />
+
+                    {props.setRetryMethods && (
+                        <>
+                            <Input
+                                placeholder="Retry Methods"
+                                disabled={true}
+                                style={{
+                                    fontWeight: "bold",
+                                    fontSize: 17,
+                                    opacity: 1,
+                                }}
+                                containerStyle={{ height: 50 }}
+                                inputContainerStyle={{
+                                    borderColor: "rgba(255,255,255,0)",
+                                }}
+                            />
+
+                            <ButtonGroup
+                                selectedIndexes={selectedMethodsIndex}
+                                // @ts-ignore
+                                onPress={setSelectedMethodsIndex}
```

Comment:
I think it might be worth pushing a PR request further upstream to fix the types, I did put in the ts-ignore but it always leads to bad things!